src/components/Link/src/index.tsx

```diff
@@ -1,68 +1,73 @@
 import React from 'react';
-import { Link as MaterialLink } from '@material-ui/core';
 import BaseProps from '@gemeente-denhaag/baseprops';
+import { SvgIconProps } from '@gemeente-denhaag/icons';
+import clsx from 'clsx';
 
-export interface LinkProps extends BaseProps, React.AnchorHTMLAttributes<never> {
-  /**
-   * The color of the link.
-   */
-  color?: 'initial' | 'inherit' | 'primary' | 'secondary' | 'textPrimary' | 'textSecondary' | 'error';
-
-  /**
-   * The component used for the root node.
-   * Either a string to use a HTML element or a component.
-   */
-  component?: React.ElementType;
+import './link.css';
 
+export interface LinkProps extends BaseProps, React.AnchorHTMLAttributes<never> {
```

Comment:
Perhaps use `extends Omit<BaseProps, 'classes' | 'otherPropYouDontWant'>` to get rid of unused properties from the baseprops interface. Check out [this line of code](https://github.com/nl-design-system/denhaag/pull/319/files#diff-6f6da8a603f347833a13728b5a02d3ced3d52edfd7e36c8e72bddc96c50f2cb8R7)